contracts/external/policies/SudoPolicy.sol

```diff
@@ -54,7 +55,7 @@ contract SudoPolicy is IActionPolicy, I1271Policy {
 
     function supportsInterface(bytes4 interfaceID) external pure override returns (bool) {
         return interfaceID == type(IActionPolicy).interfaceId || interfaceID == type(I1271Policy).interfaceId
-            || interfaceID == IActionPolicy.checkAction.selector
+            || interfaceID == IActionPolicy.checkAction.selector || interfaceID == type(IERC165).interfaceId
             || interfaceID == I1271Policy.check1271SignedAction.selector;
     }
```

Comment:
For the `SudoPolicy` there are currently checks against `IActionPolicy.checkAction.selector` and `I1271Policy.check1271SignedAction.selector`, WDYT about removing all selectors and instead making it something like this:

```solidity
function supportsInterface(bytes4 interfaceID) external pure override returns (bool) {
    return (
        interfaceID == type(IERC165).interfaceId ||
        interfaceID == type(IPolicy).interfaceId ||
        interfaceID == type(IActionPolicy).interfaceId ||
        interfaceID == type(I1271Policy).interfaceId
    );
}
```